cmd/query/app/querysvc/query_service.go

```diff
@@ -128,7 +140,7 @@ func (qs QueryService) ArchiveTrace(ctx context.Context, query spanstore.GetTrac
 }
 
 // Adjust applies adjusters to the trace.
-func (qs QueryService) Adjust(trace *model.Trace) (*model.Trace, error) {
+func (qs QueryService) adjust(trace *model.Trace) (*model.Trace, error) {
 	return qs.options.Adjuster.Adjust(trace)
```

Comment:
do any of v1 adjusters return errors?

Reply:
no they don't - should i open a separate PR to remove that from the interface?

Reply:
https://github.com/jaegertracing/jaeger/pull/6426